pallets/anchor/src/tests.rs

```diff
@@ -432,7 +430,7 @@ fn should_fail_with_when_any_byte_is_changed_in_proof() {
 				refund_value.into(),
 				commitment_element
 			),
-			crate::Error::<Test, _>::InvalidWithdrawProof
+			pallet_verifier::Error::<Test, _>::VerifyError
```

Comment:
Im getting InvalidData error here, and Im not sure that it is exactly. It comes from arkworks while verifying proof with changed bytes. I would like to investigate more if we are not in a rush

Reply:
Ok sounds good.

Reply:
It looks like it was hitting serialization error: https://github.com/arkworks-rs/algebra/blob/master/serialize/src/error.rs#L9, depending on which byte you change. I changed it back to hit the original error. But overall, more thinking is needed to understand what kinds of attack can be performed by changing the proof, and then test against that